lua/snippy/reader/snipmate.lua

```diff
@@ -27,7 +27,8 @@ local function read_snippets_file(snippets_file)
         local line = lines[i]
         local prefix = line:match('%s+(%S+)%s*')
         assert(prefix, 'prefix is nil: ' .. line .. ', file: ' .. snippets_file)
-        local description = line:match('%s*"(.+)"%s*$')
+        local description = line:match('%s*"(.+)"%s*')
+        local modifier = line:match('[bwi]?$') -- allow at most one modifier
```

Comment:
I believe the options you added are mutually exclusive, so this should be OK for now.

Reply:
Yes. That's why I also added the comment behind the statement. If this changes, the regexp needs to change as well.